ansible/roles/cartodb.resque/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+ - name: Ensure pid file dir exists
+   command: mkdir -p {{ cartodb_resque_pid_dir }}
+   sudo: False
+
+ - name: Copy resque worker job template
+   template: src=cartodb_resque_worker.conf.j2 dest=/etc/init/cartodb_resque_worker.conf
+
+ - name: Copy resque launcher template
+   template: src=cartodb_start_resque.conf.j2 dest=/etc/init/cartodb_start_resque.conf
+
+ - name: Restart resque workers
+   command: service cartodb_resque_worker restart WORKERNUM={{ item }}
```

Comment:
Interesting approach here. I'm not sure I have a much better alternative to keep this in Ansible land.